app/models/pusher.rb

```diff
@@ -94,38 +91,14 @@ def inspect
     "<Pusher #{attrs.join(' ')}>"
   end
 
-  def update_remote_bundler_api(to = RestClient)
-    return unless @bundler_api_url
-
-    json = {
-      "name"           => spec.name,
-      "version"        => spec.version.to_s,
-      "platform"       => spec.platform.to_s,
-      "prerelease"     => !!spec.version.prerelease?, # rubocop:disable Style/DoubleNegation
-      "rubygems_token" => @bundler_token
-    }.to_json
-
-    begin
-      timeout(5) do
-        to.post @bundler_api_url,
-          json,
-          :timeout        => 5,
-          :open_timeout   => 5,
-          'Content-Type'  => 'application/json'
-      end
-    rescue StandardError, Interrupt
-      false
-    end
-  end
-
   private
 
   def after_write
     @version_id = version.id
     Delayed::Job.enqueue Indexer.new, priority: PRIORITIES[:push]
     rubygem.delay.index_document
+    expire_api_memcached
     enqueue_web_hook_jobs
-    update_remote_bundler_api
```

Comment:
Can you leave all this in here for now? I want to ship this PR before we disable the bundler-api completely so we need to keep sending these updates.